eclass/libtool.eclass

```diff
@@ -25,6 +25,69 @@ esac
 
 inherit toolchain-funcs
 
+# @ECLASS_VARIABLE: LIBTOOL
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# The default libtool implementation. If unset GNU libtool will be used in most
+# cases.
+
+# @FUNCTION: eslibtool
+# @USAGE: [flag]
+# @DESCRIPTION:
+# If the LIBTOOL variable in the user's environment is set and contains one of
+# the slibtool symlinks it will be modified as required by the ebuild.
+#
+# If the MAKE or MAKEFLAGS environment variables are set they will also be
+# modified accordingly.
+#
+# The flag should '-shared', '-static' or unset to use the equivalent slibtool
+# symlink. With no arguments both shared and static libraries will be enabled.
+eslibtool() {
+	[[ -n "${LIBTOOL}" ]] || return
+
+	local d new_libtool
+	local shared="${1:-}"
```

Comment:
Does `:-` do anything here? Also, if it's supposed to be shared-or-static, then perhaps using the name `flag` or `suffix` would be better here.